main/src/ca/uwaterloo/flix/language/phase/unification/EffUnification3.scala

```diff
@@ -429,6 +429,8 @@ object EffUnification3 {
     implicit val renv: RigidityEnv = RigidityEnv.empty
     implicit val bimap: SortedBimap[Atom, Int] = mkBidirectionalVarMap(Atom.getAtoms(tpe))
 
+    implicit val alg: ZhegalkinAlgebra.type = ZhegalkinAlgebra
```

Comment:
I use .type here because ZhegalkinAlgebra is an object.

Next step is to make it into a class too.